views/course_form/views.py

```diff
@@ -18,6 +19,9 @@ def get(self, request, code: str | None = None, semester: str | None = None):
         '''
         if not self.__can_use_form(request.user, code, semester):
             return redirect(reverse("home"))
+
+
```

Comment:
remove blank lines. At most there should be one blank line between statements in functions